src/ASM/ASMs3D.C

```diff
@@ -2113,7 +2115,7 @@ bool ASMs3D::integrate (Integrand& integrand,
 #endif
 
           // Cartesian coordinates of current integration point
-          X = Xnod * fe.N;
+          X.assign(Xnod * fe.N);
```

Comment:
Here too, can use `X.u = itgPts[iel][ip].data()` instead of using param temporary and changing to assign.

Reply:
done.